go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/tarantool/go-tupleconv
+
+go 1.19
+
+require (
+	github.com/google/uuid v1.3.0
+	github.com/shopspring/decimal v1.3.1
+	github.com/stretchr/testify v1.7.1
+	github.com/tarantool/go-tarantool/v2 v2.0.0-20230630180713-b30e2b69c9fc
```

Comment:
Is v2 already released?

Reply:
No, I used this version for completeness of `tarantool.Field` (since https://github.com/tarantool/go-tarantool/commit/40fe0d3a6926559328bf4e1cd93d4c90a5b316a4 was not released), but introduced own structure. So now released version is used.